apps/designer/app/designer/features/style-panel/sections/spacing/value-text.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import React from "react";
+import { SpacingLayout } from "./layout";
+import { ValueText as ValueTextCommponent } from "./value-text";
+
+const Template = (
+  args: Pick<
+    React.ComponentProps<typeof ValueTextCommponent>,
+    "source" | "value"
+  >
+) => {
+  const [hovered, setHovered] = React.useState<{ property: string }>();
+  return (
+    <SpacingLayout
+      onHover={setHovered}
+      onClick={() => null}
+      renderCell={({ property }) => (
+        <ValueTextCommponent
+          isActive={property === hovered?.property}
+          {...args}
+        />
+      )}
+    />
+  );
+};
+
+export default {
+  title: "ValueText",
+  component: Template,
+  argTypes: {
+    source: { control: "select", options: ["set", "unset", "inherited"] },
+    value: {
+      control: "select",
+      options: {
+        "0": { type: "unit", value: 0, unit: "number" },
```

Comment:
probably use [parseCssValue](https://github.com/webstudio-is/webstudio-designer/blob/e78919cc999320f2d42d65e4ee43605d1a308966/apps/designer/app/designer/features/style-panel/shared/parse-css-value.ts#L28) not a big deal anyway